aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -463,15 +463,20 @@ void beepidiboop(const at::Tensor& input, const at::Tensor& output) {
   auto num_pixels_output = xout * yout;
 
   UINT32* unpacked_input_p = (UINT32*)malloc(sizeof(UINT32) * num_pixels_input);
-  const uint8_t* packed_input_p = (const uint8_t*)input.data_ptr<uint8_t>();
-  unpack_rgb((uint8_t*)unpacked_input_p, packed_input_p, num_pixels_input);
 
-  UINT32* unpacked_output_p =
-      ImagingResampleInner(unpacked_input_p, xin, yin, xout, yout);
+  for (const auto i : c10::irange(input.size(0))) {
```

Comment:
Code is largely unchanged, I just put it under this loop over `batch_size`

Also forgot to call `free()` below.